gengokit/generator/gen_test.go

```diff
@@ -116,7 +115,7 @@ func svcMethodsNames(methods []*svcdef.ServiceMethod) []string {
 	return mNames
 }
 
-func stringToTemplateExector(def, importPath string) (*gengokit.TemplateExecutor, error) {
+func stringToTemplateExector(def, importPath string) (*gengokit.Executor, error) {
```

Comment:
I don't any attachment to the name, I thought I was following a pattern in the standard library calling it Executor but I was mistaken.

The [`text/templates`](https://golang.org/pkg/text/template/#Template.Execute) calls this value the `data`.  

```
func (t *Template) Execute(wr io.Writer, data interface{}) error
```

So my vote would be for `gengokit.Data`, but if you feel strongly about `gengokit.Values` I would be fine with that too.